spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/DeleteOrphanFilesSparkAction.java

```diff
@@ -326,6 +344,96 @@ private static FlatMapFunction<Iterator<String>, String> listDirsRecursively(
     };
   }
 
+  @VisibleForTesting
+  static List<String> findOrphanFiles(SparkSession spark,
```

Comment:
This method currently uses the preferred formatting in Iceberg but I see that other methods in this class use other formatting cause the names are long. I don't have a preference. Up to you if you want to make it consistent with other methods.

<img width="619" alt="image" src="https://user-images.githubusercontent.com/6235869/179638774-f49a50a8-f22e-47cb-a318-2008ca5c0b95.png">

Reply:
formatted like above